### PR TITLE
Add batch read of parquet file from S3

### LINKED ISSUE #2390: Document: feat(batch): support batch read s3 parquet file
This issue tracks the documentation update needed for the merged PR #17673.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/17673
Source PR Merged At: 2024-07-18T15:49:07Z

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**

  - [ What's changed? Which parts of the docs are affected? ]

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - https://github.com/risingwavelabs/risingwave/pull/17673/files

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave/pull/17673

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/ingest/ingest-from-s3.md [hunk 1]
```diff
@@ -216,6 +216,28 @@ CREATE MATERIALIZED VIEW mv AS SELECT * FROM s3_source;
...
+SELECT * FROM file_scan(
+  'parquet',
+  's3',
+  'ap-southeast-2',
+  'xxxxxxxxxx',
+  'yyyyyyyy',
+  's3://your-bucket/path/to/fila_name.parquet'
+);
+
+----RESULT
```
Comment (ShanlanLi) [inline docs/ingest/ingest-from-s3.md:237]: @chenzl25 Could you help provide a result here? So that users can know how the output look like.

Comment (chenzl25) [reply docs/ingest/ingest-from-s3.md:237]: Sure
```
 a | b | c
---+---+---
 2 | 2 | 3
 4 | 5 | 6
(2 rows)
```

Comment (ShanlanLi) [reply docs/ingest/ingest-from-s3.md:237]: Thanks! I've added the result, please help review it.

**Review by chenzl25 — APPROVED** [review]
LGTM

**Review by WanYixian — APPROVED** [review]

**Pull request merged**